Jenkinsfile

```
@@ -40,5 +40,5 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipeline(['distros' : ['centos7', 'centos8', 'leap15']])
+packageBuildingPipeline(['distros' : ['el8', 'leap15']])
```

I think given that we are now not building this for CentOS 7 any more, it would be prudent to use the function that does a build+test on all of our release branches as well as adding some validation tests for mpifileutils in the form of test-tags:

```suggestion
packageBuildingPipelineDAOSTest(['distros' : ['el8', 'leap15']
                                 'test-tag': 'some-test-tag(s)'])
```

@daltonbohning Could you suggest some sanity/smoke-test mpifileutils test tags that @wiliamhuang could put in place of `some-test-tag(s)` above that would help validate this update by running some functional tests with the new mpifileutils in this PR?

---

@brianjmurrell @wiliamhuang 
You can use `mpifileutils` for all tests using `mpifileutils`.
A simplified set that I think would cover just this update: `dm_posix_types dm_serial_small dm_dst_create dm_posix_meta_entry`

---

@brianjmurrell @daltonbohning Thank you very much!

---

Is GitHub having some problem or are the above requested changes not in this PR yet?